Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -353,7 +347,32 @@ namespace O3DE::ProjectManager
         tags.reserve(gems.size());
         for (const QModelIndex& modelIndex : gems)
         {
-            tags.push_back({ GemModel::GetDisplayName(modelIndex), GemModel::GetName(modelIndex) });
+            const GemInfo& gemInfo = GemModel::GetGemInfo(modelIndex);
+            if(gemInfo.m_isEngineGem)
```

Comment:
You cannot change engine gem versions like external gems,  there is only one version of an engine gem available because it comes with the engine.  Currently all of our engine gems are only assumed compatible with the engine they come with.

You cannot, for example, select a different version of the Atom gem, and normally you don't want to, when you register your project with a new engine, that's when you address compatibility for all the engine gems, unlike remote/external gems that can be updated whenever and may support many engine versions.